src/sonic_ax_impl/mibs/ietf/rfc3433.py

```diff
@@ -260,18 +382,23 @@ def reinit_data(self):
         self.ent_phy_sensor_value_map = {}
         self.ent_phy_sensor_oper_state_map = {}
 
-        transceiver_dom_encoded = Namespace.dbs_keys(self.statedb, mibs.STATE_DB,
-                                                    self.TRANSCEIVER_DOM_KEY_PATTERN)
+        transceiver_dom_encoded = Namespace.dbs_keys(self.statedb, mibs.STATE_DB, self.TRANSCEIVER_DOM_KEY_PATTERN)
         if transceiver_dom_encoded:
             self.transceiver_dom = [entry for entry in transceiver_dom_encoded]
 
-    def update_data(self):
-        """
-        Update sensors cache.
-        """
+        fan_sensor_encoded = Namespace.dbs_keys(self.statedb, mibs.STATE_DB, self.FAN_SENSOR_KEY_PATTERN)
```

Comment:
Namespace.dbs_keys() will get keys from all namespace DBs.
PSU/Fan/thermal sensor data will be specific to host namespace DB , and not per-asic data. Is that correct?
If that is the case, instead of getting data from Namespace.dbs_keys(),  you can get data from just host DB as this will avoid iterating through all namespace DBs.
could be done 2 ways:
1.         self.host_statedb = mibs.init_db()
         self.host_statedb.connect(mibs.STATE_DB)
 this will return connector of host namespce DB; and use this to get keys and get_all operation.
2. or use the first db connector from existing  list: self.statedb[0].

Reply:
revised, using the second way.